cmd/vela-terraform/init.go

```diff
@@ -139,13 +138,11 @@ func (i *Init) Command(dir string) *exec.Cmd {
 
 	// check if PluginDirs is provided
 	if len(i.InitOptions.PluginDirs) > 0 {
-		var configs string
 		for _, v := range i.InitOptions.PluginDirs {
-			configs += fmt.Sprintf("-plugin-dir=%s ", v)
+			flag := fmt.Sprintf(`-plugin-dir=%s`, v)
+			// add flag for PluginDirs from provided init command
+			flags = append(flags, flag)
```

Comment:
Do you think we could condense this into one line?

```suggestion
			// add flag for PluginDirs from provided init command
			flags = append(flags, fmt.Sprintf(`-plugin-dir=%s`, v))
```